### PR TITLE
Implement sort_complex for JAX frontend

### PR DESCRIPTION
**Pull request by H4ckOm**

### CODE REVIEW

**Comment by H4ckOm** [comment]
Close #9959

File: ivy/functional/frontends/jax/numpy/searching_sorting.py [hunk 1]
```diff
@@ -83,3 +83,8 @@ def nanargmin(a, /, *, axis=None, out=None, keepdims=None):
     a = ivy.where(nan_mask, ivy.inf, a)
     res = ivy.argmin(a, axis=axis, keepdims=keepdims)
     return ivy.where(ivy.all(nan_mask, axis=axis, keepdims=keepdims), -1, res)
+
+
+@to_ivy_arrays_and_back
+def sort_complex(a):
+    return ivy.sort_complex(a)
```
Comment (ZoeCD) [inline ivy/functional/frontends/jax/numpy/searching_sorting.py:90]: Hi! The problem is that `ivy.sort_complex` does not exist yet. You can implement it as a composition. I recommend reading [this guide](https://lets-unify.ai/ivy/deep_dive/ivy_frontends.html#compositions) and [this one](https://lets-unify.ai/ivy/contributing/open_tasks.html#frontend-apis) to get a better understanding of how to handle this situation. If you have any questions, please let me know! 😄

**Review by ZoeCD — CHANGES_REQUESTED** [review]

File: ivy/functional/frontends/jax/numpy/searching_sorting.py (hunk 1 above)
Comment (H4ckOm) [reply ivy/functional/frontends/jax/numpy/searching_sorting.py]: Hi! I have updated this function as a composition of pre-implemented ivy functions. Please can you check again?

Comment (H4ckOm) [reply ivy/functional/frontends/jax/numpy/searching_sorting.py]: @ZoeCD

Comment (ZoeCD) [reply ivy/functional/frontends/jax/numpy/searching_sorting.py]: Hi! There seem to be some problems with the implementation. The `intelligent-test` are showing the following exception `hypothesis.errors.InvalidArgument: Cannot sample from a length-zero sequence.` And `test-frontend-jax` is showing `TypeError: can't convert complex to float`. Could you check that, please? 🙌

**Comment by ivy-seed** [comment]
This PR has been labelled as stale because it has been inactive for more than 7 days. If you would like to continue working on this PR, then please add another comment or this PR will be closed in 7 days.

**Comment by ivy-seed** [comment]
This PR has been closed because it has been marked as stale for more than 7 days with no activity.

**Comment by H4ckOm** [comment]
Can we reopen this?

**Pull request closed without merging**